bears/stylus/StylintBear.py

```diff
@@ -0,0 +1,210 @@
+            ``$`` when defining a variable is optional, but is a good idea
+            if you want to prevent ambiguity. Not including the ``$`` sets up
+            situations where you wonder: "Is this a variable or a value?"
+            For instance: padding ``$default`` is easier to understand than
+            padding ``default``.
+        :param allow_semicolons:
+            Enforce or disallow semicolons.
+        :param sort_order:
+            Enforce a particular sort order when declaring properties. Throws
+            a warning if you don't follow the order. If set to ``false``, allow
+            any order.
+        :param allow_stacked_properties:
+            No one-liners. Enforce putting properties on new lines.
+        :param check_valid_property:
+            Check that a property is valid CSS or HTML.
+        :param allow_zero_units:
+            Looks for instances of ``0px``. You don't need the px. Checks all
+            units, not just px.
+        :param z_index_normalize:
+            Enforce some (very) basic z-index sanity. Any number passed in
+            will be used as the base for your z-index values. Throws an error
+            if your value is not normalized.
+        """
+        options = {
+            'blocks': allow_block_keyword,
+            'brackets': allow_brackets,
+            'colons': allow_colons,
+            'colors': allow_colors,
+            'commaSpace': allow_spaces_after_commas,
+            'commentSpace': allow_spaces_after_comments,
+            'cssLiteral': allow_css_literal,
+            'depthLimit': max_selector_depth,
+            'duplicates': check_duplicates,
+            'efficient': allow_efficient_properties,
+            'extendPref': set_extend_preference,
+            'indentPref': indent_size,
+            'leadingZero': allow_leading_zero,
+            'maxErrors': set_max_errors,
+            'maxWarnings': set_max_warnings,
+            'mixed': mixed_spaces_and_tabs,
+            'namingConvention': set_naming_convention,
+            'namingConventionStrict': set_strict_naming_convention,
+            'none': allow_none_keyword,
+            'noImportant': check_no_important_keyword,
+            'parenSpace': allow_spaces_inside_parens,
+            'placeholders': allow_placeholder,
+            'prefixVarsWithDollar': prefix_vars_with_dollar,
+            'semicolons': allow_semicolons,
+            'sortOrder': sort_order,
+            'stackedProperties': allow_stacked_properties,
+            'trailingWhitespace': allow_trailing_whitespace,
+            'valid': check_valid_property,
+            'zeroUnits': allow_zero_units,
+            'zIndexNormalize': z_index_normalize,
+            'groupOutputByFile': True}
+
+        options['reporterOptions'] = {
+            'columns': ['lineData', 'severity', 'description', 'rule'],
+            'columnSplitter': '  ',
+            'showHeaders': False,
+            'truncate': True
+        }
```

Comment:
wanna inline that directly into the dict above?
```
options = {
    '...': ...,
    'reporterOptions': {
        'columns': [...]
        ...
```
I think that makes it easier to understand, as there's more data to read instead of code ;)